epub33/epub-a11y-eaa-mapping/index.html

```diff
@@ -88,21 +87,30 @@ <h2>How EPUB conforming to EPUB accessibility guidelines cover European Accessib
 
 			<ul>
 				<li>discoverability of accessibility features of publications;</li>
-				<li>evaluation and certification of accessible EPUB publications.</li>
+				<li>conformance to Web Content Accessibility Guidelines;</li>
+				<li>specific EPUB requirements for accessibility;</li>
+				<li>evaluation and conformance reporting of accessible EPUB publications.</li>
 			</ul>
 
 			<p>Providing accessibility metadata facilitates informed decisions about the usability of an EPUB
 				publication. Consumers can examine the characteristics of the content and decide whether an EPUB
 				publication is appropriate for each need.</p>
 
-			<p>While it has always been possible to create EPUB publications with a high degree of accessibility, this
-				document also establishes formal requirements for contents to be certified as accessible. These
+			<p>While it has always been possible to create EPUB publications with a high degree of accessibility, 
+				this document establishes the formal accessibility requirements that content must meet to be declared 
+				as conformant with the EPUB Accessibility. These
 				requirements provide EPUB creators with a clear set of guidelines for evaluating their content with
-				respect to quality certification and enable it.</p>
+				respect to quality assurance and enable it.</p>
 
 			<p>The EPUB Accessibility does not target a single version of EPUB (most commercially available publications
 				are in EPUB 3 or EPUB 2 format). It is designed to be applicable to all EPUB publications conforming to
 				any version or profile, including future versions of the standard.</p>
+			
+			<p>The EPUB format provides the ability to create reflowable publications (whose content adapts to the user's viewport,
+				depending also on the user's chosen graphical characteristics, such as font size, line spacing, etc.), but it also
+				allows the creation of EPUB Fixed Layout in which each element is positioned absolutely on the page (mainly used
+				for publications such as comics, illustrated children’s books and art books). Specific features of Fixed Layout EPUBs
+				should be considered in the light of all applicable accessibility requirements.</p>
```

Comment:
This is a long sentence. Maybe break it up like this (also offering a possible rewrite for the last sentence, but not sure if that's exactly what you're after, or if it should be in a note, so feel free to ignore!):

```suggestion
			<p>The EPUB format provides the ability to create reflowable publications whose content adapts to the user's viewport, including the user's chosen graphical characteristics, such as font size, line spacing, etc. It also
				allows the creation of fixed layout content in which each element is positioned absolutely on the page (mainly used
				for publications such as comics, illustrated children’s books and art books). Specific features of fixed layouts (e.g., zooming of text) present challenges for accessibility, however. Authors need to consider these challenges and how best they can mitigate them before employing fixed layouts. Refer to the EPUB Fixed Layout Guidelines for more information.</p>
```